### DISCUSSION TITLE
[待讨论] [ 跨平台 - 移动端需求 ] A suggestion

### DISCUSSION THREAD

**Discussion by 00000000002** [GeekLee2012/Less-Player-Desktop#16] [Ideas]
Realize cross platform adaptation as soon as possible, such as Android system, IOS system, tablet, etc

**Comment by GeekLee2012** [comment]
Sorry，我就不用英语交流啦，相信朋友你能看懂中文，哈哈。即使看不懂的话，也会有在线翻译。

对于跨平台、跨设备的需求，我目前没有太多时间精力去开发。
移动端开发，我并不擅长；想进行开发的话，需要先重新学习。
开发测试，也需要诸多相应的硬件设备。因为，模拟器并不可能完全替代真机。

开源项目，抛砖引玉，不可能尽善尽美。
许多需求问题，留待其他开发者朋友们一起来参与解决吧。

最后，提醒一下：
由于播放器涉及版权数据，移动端版本，更有可能被拥有版权的官方出手“制裁”。
